dev/src/Command/ReleaseInfoCommand.php

```diff
@@ -63,42 +71,110 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $github = new Github(
             new RunShell(),
             $this->httpClient,
-            $input->getOption('token')
+            $input->getOption('token'),
+            $output
```

Comment:
The constructor in Github class doesn't do anything with the `$output` argument. What's the role of this `$output`?

Reply:
Yes it does, the syntax `private ?OutputInterface $output` in a constructor sets it to a private property. It's new syntax sugar in PHP 8.0. See [PHP Constructor Property Promotion](https://stitcher.io/blog/constructor-promotion-in-php-8).